src/codegen/sys/tests.rs

```diff
@@ -381,9 +381,11 @@ fn pkg_config_cflags(packages: &[&str]) -> Result<Vec<String>, Box<dyn Error>> {
     let mut cmd = Command::new(pkg_config);
     cmd.arg("--cflags");
     cmd.args(packages);
+    cmd.stderr(Stdio::inherit());
     let out = cmd.output()?;
     if !out.status.success() {
-        return Err(format!("command {cmd:?} returned {}", out.status).into());
+        let (status, stdout) = (out.status, str::from_utf8(&out.stdout)?);
```

Comment:
Should this (and below) maybe use `from_utf8_lossy()`? Some error might cause non-UTF8 to be printed and it would seem useful to print whatever can be printed of that

Reply:
Great idea!